Makefile

```diff
@@ -156,10 +153,23 @@ build-test-e2e: build-services
 build-python-wheels:  ## Build all Python wheels
 	./pants filter --target-type=python_distribution :: | xargs ./pants package
 
-.PHONY: build-services
-build-services: graplctl lambdas build-python-wheels ## Build Grapl services
+.PHONY: build-python-lambda-zips
+build-python-lambda-zips:
+	./pants filter --target-type=python_awslambda :: | xargs ./pants package
+
+.PHONY: build-python-pants
+build-python-pants: build-python-wheels build-python-lambda-zips
```

Comment:
Maybe `build-python-with-pants` is better?

Reply:
I'm wondering if we should just include graplctl build into this target.

Reply:
🤔 Does it need to include "pants" at all anymore?

It's somewhat like having a `build-rust-with-cargo`, `compile-c-with-gcc`, or `build-jar-with-maven` targets.

Reply:
I think including "pants" is helpful because it looks like we're building sources with both Pants and Docker, and I think this helps clarify that. The new name I made for Docker could better match this Pants one, how about we keep `build-with-pants` and `build-docker-services` becomes `build-with-docker`?

Reply:
Sounds good.

I look forward to the day when we can hopefully just have a `build` target 😂

Reply:
Maybe today is that day.